src/js/pagination.js

```diff
@@ -0,0 +1,109 @@
+document.addEventListener("DOMContentLoaded", async () => {
+  setupPagination(fetchAndRenderExercises);
+});
+
+import api from "./api";
+
+async function fetchAndRenderExercises(page) {
+  const data = await getExercises(page);
+  console.log("Current Page Data:", data.results);
+
+}
+
+let firstBtn;
+let prevBtn;
+let nextBtn;
+let lastBtn;
+let pageInfo;
+
+export const getContentPagination = () => {
+  firstBtn = document.getElementById("first-btn");
+  prevBtn = document.getElementById("prev-btn");
+  nextBtn = document.getElementById("next-btn");
+  lastBtn = document.getElementById("last-btn");
+  pageInfo = document.getElementById("page-info");
+};
+
+async function setupPagination(callback) {
+  getContentPagination();
+
+
+  const initialData = await getExercises(1);
+  const totalPages = initialData.totalPages;
```

Comment:
For now, let's leave it here, the number of pages should be a parameter for pagination function